### PR TITLE
Add API key to header in SDKs

### PR DESCRIPTION
**Pull request by jakubno**

### CODE REVIEW

File: openapi.yml
Comment (jakubno) [inline openapi.yml:1]: Should I add this? I will maybe setup git submodule from API repo, wdyt?

**Review by jakubno — COMMENTED** [review]
Add python-dotenv to dependencies

File: openapi.yml
Comment (ValentaTomas) [reply openapi.yml:1]: I would go with git subtree here:
- It behaves more like ordinary git files and it looks simpler to me
- The api repo is private so people may not have access to the yaml file with submodules

**Comment by ValentaTomas** [comment]
I think we should be merging this to the `infra-update` branch for now and not to `main`.

**Comment by ValentaTomas** [comment]
I'm also not sure if it should be `Authorization`: `Bearer ...` header (which I think is more for *user* access tokens) and not a simple `X-API-KEY` (https://swagger.io/docs/specification/authentication/api-keys/). If we are using `Authorization` header the `Basic ...` could probably be used (https://blog.stoplight.io/api-keys-best-practices-to-authenticate-apis)?

**Pull request closed without merging**